addons/pos_razorpay/models/razorpay_pos_request.py

```diff
@@ -29,10 +33,9 @@ def _call_razorpay(self, endpoint, payload):
         :return The JSON-formatted content of the response.
         :rtype: dict
         """
-        endpoint = f'{self._razorpay_get_endpoint()}{endpoint}'
-        request_timeout = self.payment_method.env['ir.config_parameter'].sudo().get_param('pos_razorpay.timeout', REQUEST_TIMEOUT)
+        endpoint = f'{self._razorpay_get_endpoint(endpoint)}{endpoint}'
```

Comment:
the variable names need to be adapted, we are using `endpoint` after `endpoint` and assigning it to `endpoint`